apps/blockchain/ethereum/src/Bridge.sol

```diff
@@ -370,6 +370,8 @@ contract Starklane is IStarklaneEvent, UUPSOwnableProxied, StarklaneState, Stark
         snaddress collectionL2,
         bool force
     ) external onlyOwner {
+            require( collectionL1 != address(0x0), "Invalid L1 address");
```

Comment:
Sorry I miss this part in my previous review.
Could you please use solidity custom errors instead as other error:
```solidity
error BridgeNotEnabledError();
// ....

        if (!_enabled) {
            revert BridgeNotEnabledError();
        }
```